### PR TITLE
A new user is now greeted with a startup page

### PR DESCRIPTION
**Pull request by udeyrishi**
- This is probably the wrong way of doing MVC, but I'm keeping it the same way as BrowseInventoryFragment
- There is some issue with no_friends_layout.xml. The pull down refresh circle kinda stutters. Does someone know how to fix it?

### CODE REVIEW

**Comment by kyleoshaughnessy** [comment]
Does this cause any existing tests to fail? if not im all good with this.

**Comment by MEGASTACK** [comment]
looking...

**Comment by udeyrishi** [comment]
The tests on master for Browse Inventories are in a pretty horrible state already

**Comment by MEGASTACK** [comment]
@udeyrishi I think this is acceptable

**Comment by MEGASTACK** [comment]
@udeyrishi WAIT I found a problem

File: T03/app/src/main/java/ca/ualberta/cmput301/t03/inventory/BrowseInventoryFragment.java [hunk 1]
```diff
@@ -338,21 +307,35 @@ protected Void doInBackground(Void... params) {
...
+                    }
+                    else {
+                        if (mSwipeRefreshLayout.isRefreshing()) {
+                            mSwipeRefreshLayout.setRefreshing(false);
+                        }
+                        adapter.notifyUpdated(model);
+                    }
+                } else {
+                    synchronized (uiLock) {
+                        getFragmentManager().beginTransaction()
```
Comment (MEGASTACK) [inline T03/app/src/main/java/ca/ualberta/cmput301/t03/inventory/BrowseInventoryFragment.java]: NPE here

**Comment by MEGASTACK** [comment]
I think the pull down refresh thing is supposed to be used with list views, if you use it with something else it might do strange things

**Comment by udeyrishi** [comment]
I'm using it with top traders. It's a group view there (or something else, not a list brow though)

**Comment by MEGASTACK** [comment]
I think it could be null when you navigate away from that tab before it has loaded. I think its generally not a good idea to do fragment swapping in asynchronous task callbacks

**Comment by udeyrishi** [comment]
That's a good point. But I think it's okay in this case because it's in a  synchronized block

**Comment by MEGASTACK** [comment]
@udeyrishi: Does that actually make a difference? getFragmentManager() in a support v4 fragment will return the support v4 fragmentManager

**Comment by MEGASTACK** [comment]
The only reason I think that it might is if the application lifetime is longer then the fragment (and somehow the fragment got destroyed before the asynctask callback was called)

**Comment by udeyrishi** [comment]
Maybe, doesn't hurt to have explicit support fragment call though. More important is the change in import statements

**Comment by MEGASTACK** [comment]
Good call. (though, actually I think all 3 of those fragments can be deleted... afaik we don't use friend search at all, and we just use the layouts for the item stuff)

**Comment by udeyrishi** [comment]
Contents already merged in through q-ui-branch

**Pull request merged**